build.js

```diff
@@ -64,11 +64,7 @@
             name: 'global-deps',
```

Comment:
d'oh forgot it also has a js component even though the css doesn't have output anymore